### PR TITLE
Added find_mac and find_ip action and aliases

### PR DESCRIPTION
**Pull request by jcoeder**
Added find_mac and find_ip action and aliases

### CODE REVIEW

File: actions/find_ip.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from lib.aci import ACIBaseActions
+
+
+class findIP(ACIBaseActions):
```
Comment (nmaludy) [inline actions/find_ip.py]: Class names should start with a Capital Letter

File: actions/find_ip.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    def run(self, apic="default", credentials=None, ip=None):
```
Comment (nmaludy) [inline actions/find_ip.py]: the apic field is marked as `required: true` in the action metadata, yet you set it to a default string here in the python, need to pick one. Either set a default in the action metadata or remove the default in the python file. Similarly, if `ip` is marked required in the action metadata, you should remove the default here in the python file.

File: README.md [hunk 3]
```diff
@@ -41,12 +41,19 @@ the `credentials` parameter. This allows for an audit trail within ACI.
 * `cisco_aci.create_tenant` - Create a Tenant
 * `cisco_aci.get_snapshots` - Retrieve all snapshots
 * `cisco_aci.create_snapshot` - Create a snapshot
+* `cisco_aci.find_mac` - Find a MAC address in the fabric
```
Comment (nmaludy) [inline README.md:44]: Missing a changelog entry in CHANGES.md

File: actions/find_ip.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```
Comment (nmaludy) [inline actions/find_ip.py:1]: Missing unit tests for new python code.

**Review by nmaludy — CHANGES_REQUESTED** [review]
Couple small changes. There are some examples of unit tests in the tests directory.

File: pack.yaml [hunk 5]
```diff
@@ -1,11 +1,12 @@
 ref: cisco_aci
 name: cisco_aci
 description: Cisco ACI pack
-version: 0.1.6
+version: 0.1.7
```
Comment (arm4b) [inline pack.yaml]:
```suggestion
version: 0.2.0
```

Considering this PR is not a bugfix, but a feature with new changes, please bump the version accordingly to https://semver.org/#summary

**Comment by jcoeder** [comment]
Made recommended changes.  Tests are very basic - based on existing tests.

File: tests/test_action_find_ip.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+# See the License for the specific language governing permissions and
+
+# from mock import MagicMock
+
+from aci_base_action_test_case import CiscoACIBaseActionTestCase
+
+from find_ip import FindIP
+
+
+class GetAPsActionTestCase(CiscoACIBaseActionTestCase):
```
Comment (nmaludy) [inline tests/test_action_find_ip.py:22]: This is good boiler plate, but now you need some actual test functions that perform tests on your code.

**Review by nmaludy — CHANGES_REQUESTED** [review]
Getting better! Just need some actual tests and you'll be good!

**Comment by nmaludy** [comment]
I see why you're confused on the tests, because the ones in this pack are bad examples and don't actually test anything. Let me try to find an example from another pack.

**Comment by nmaludy** [comment]
@jcoeder here is a good example https://github.com/StackStorm-Exchange/stackstorm-vsphere/blob/master/tests/test_action_get_tags_on_object.py#L30

There are a couple other good ones in that pack.

The general idea is: 
- Create an instance of your action
- Mock out any functions that will make a connection over a network. You setup these mocks to return fake data instead of going over network.
- Run your function with some test data
- Compare the results you get back with what you think you should expect knowing what the function is supposed to do.

Your test functions should be named with `test_` at the beginning of the name, the testing framework will auto discover these and execute them.

Here is a good intro on the `unittest` framework that we use along with `nosetest`: https://realpython.com/python-testing/

**Review by nmaludy — APPROVED** [review]
LGTM!

**Pull request merged**